README.md

```diff
@@ -22,6 +22,96 @@ GCE_PARAMS = ('...@developer.gserviceaccount.com', '/path/to/gce_account.json')
 GCE_KEYWORD_PARAMS = {'project': 'project_id'}
 ```
 
+* [GnuPG](#setting-up-gpg-encrypted-vault-password-support)
+
+#### Setting up GPG-encrypted vault-password support
+
+You will need to have setup [gpg-agent](https://www.gnupg.org/) on your computer before you start.
+
+##### Apple specific
+
+Install the latest [GPG Tools Suite for MacOX](https://gpgtools.org/)
+
+```
+brew install pwgen
+brew install gpg
+brew install gpg-agent
+```
+
+##### Ubuntu specific
+
+Install the [GNU Privacy Guard encryption suite](https://www.gnupg.org/):
+
+```
+sudo apt-get update
+sudo apt-get install pwgen
+sudo apt-get install gnupg2
+sudo apt-get install gnupg-agent
+sudo apt-get install pinentry-curses
+```
+
+##### Common
```

Comment:
Is there any third-party documentation that we can link to instead of having it here? Makes our README very long and we might not always have the most up-to-date information. Not a deal breaker though.